egs/aspire/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -21,29 +23,20 @@ set -e
 local/multi_condition/check_version.sh || exit 1;
 
 mkdir -p exp/nnet3
-if [ $stage -le 0 ]; then
-  # prepare the impulse responses
-  local/multi_condition/prepare_impulses_noises.sh --log-dir exp/make_reverb/log \
-    --db-string "$db_string" \
-    --download-rirs $download_rirs \
-    --RIR-home $RIR_home \
-    data/impulses_noises || exit 1;
-
-  # Generate the rir_list and noise_list for the reverberate_data_dir.py to corrupt the data
-  # this script just assumes air rwcp rvb2014 databases
-  python local/multi_condition/aspire_prep_rir_noise_list.py data/impulses_noises data/impulses_noises/info
-
-  # Download Musan and generate the necessary noise_list file
-  wget --no-check-certificate http://www.openslr.org/resources/17/musan.tar.gz
-  tar -zxf musan.tar.gz
-  python local/multi_condition/musan_prep_noise_list.py  --sampling-rate 16000  musan/noise/free-sound musan/noise/free-sound/noise_list
-
-  # Download the simulated RIRs
-  wget --no-check-certificate http://www.openslr.org/resources/26/sim_rir.zip
-  unzip sim_rir.zip
-fi
-
 if [ $stage -le 1 ]; then
+  if [ "$base_rirs" == "simulated" ]; then
+    # Download the package that includes the real RIRs, simulated RIRs, isotropic noises and point-source noises
```

Comment:
For the `else` condition do not prepare the impulses and noises from scratch, just download the entire rirs_noises.zip and use the lists corresponding to the real_rirs and isotropic_noises. All I wanted to ensure was that we could replicate the ASpIRE submission results, the scripts need not be the same.

Please provide the scripts, in comments, which is used to create the upload. Rather than deleting the scripts which create the which create the RIR and noise list files move them to the directory  `local/multi_condition/impulses_noises` (IIRC). Also add a note here that this process is affected by #966  and this is being resolved in #981 .